pipelines/tests/test_execution.py

```diff
@@ -58,6 +58,10 @@ async def test_step_execution_with_logging(
     assert step_exec_mock.call_count == 1
 
 
+@pytest.mark.skip(
+    "Test should be fixed - it 'blinks'. "
+    "It passes when run separately, but fails when all tests are run."
+)
```

Comment:
May you create an issue for this?

Reply:
For sure.
I will create an issue right after merging to main.
(They will be marked 'skip' in 'main' branch and it would be easier to reffer to them